### PR TITLE
Set relative bucket location

### LINKED ISSUE #121: s3 bucket path in constants should correspond to staging or production

### PR DESCRIPTION
**Pull request by lxu213**
Fixes #121

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-// export const s3BucketUrl = 'https://idd-dashboard-runs.s3.amazonaws.com/json-files/';
-export const s3BucketUrl = 'https://idd-dashboard-runs-staging.s3.amazonaws.com/json-files/';
+let dataLocation;
+if (process.env.NODE_ENV === 'development') {
+    dataLocation = 'https://idd-dashboard-runs-staging.s3.amazonaws.com/json-files/';
+} else { 
+    // indicate key prefix of relative bucket (could be staging or production)
+    dataLocation = 'json-files/';
+}
+export const s3BucketUrl = dataLocation;
```
Comment (lxu213) [inline src/utils/constants.js:10]: @BenjiLee turns out react has a very nifty `process.env.NODE_ENV` that tells you if you're in dev or production mode. Yay! Is this syntax messy?

Comment (BenjiLee) [reply src/utils/constants.js:10]: Can use this syntax as well
```
const dataLocation = process.env.NODE_ENV === 'development' ?
            'https://idd-dashboard-runs-staging.s3.amazonaws.com/json-files/'
            : 'json-files/';
```

Comment (BenjiLee) [reply src/utils/constants.js:10]: I wonder if you can just add the prefix in this code

Comment (lxu213) [reply src/utils/constants.js:10]: Oh yes, this is much cleaner!

**Pull request merged**